test/scripts/build-suggestion-bo.sh

```diff
@@ -39,3 +39,6 @@ cd ${GO_DIR}
 
 cp cmd/suggestion/bayesianoptimization/Dockerfile .
 gcloud container builds submit . --tag=${REGISTRY}/${REPO_NAME}/suggestion-bayesianoptimization:${VERSION} --project=${PROJECT}
+
+cp cmd/suggestion/cma/Dockerfile .
```

Comment:
Please create a new script `build-suggestion-cma.sh` and register it to `/test/workflows/components/workflows.libsonnet`.